encoding/thrift/outbound.go

```diff
@@ -206,7 +207,7 @@ func (c thriftClient) buildTransportRequest(reqBody envelope.Enveloper) (*transp
 		Caller:    c.cc.Caller(),
 		Service:   c.cc.Service(),
 		Encoding:  Encoding,
-		Procedure: procedureName(c.thriftService, reqBody.MethodName()),
+		Procedure: procedure.ToName(c.thriftService, reqBody.MethodName()),
```

Comment:
This isn't for thrift, this is why it's abstracted out (`procedure` is a new package on this diff) - both thrift and protobuf use it. Before, it was a private method in thrift.